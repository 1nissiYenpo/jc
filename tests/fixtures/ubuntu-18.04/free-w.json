[{"type": "Mem", "total": 8053804, "used": 1262704, "free": 5830864, "shared": 60576, "buffers": 61596, "cache": 898640, "available": 6483996}, {"type": "Swap", "total": 2097152, "used": 0, "free": 2097152}]
