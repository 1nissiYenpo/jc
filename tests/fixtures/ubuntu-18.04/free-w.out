              total        used        free      shared     buffers       cache   available
Mem:        8053804     1262704     5830864       60576       61596      898640     6483996
Swap:       2097152           0     2097152
